### PR TITLE
MWPW-135782 | Removing min height from mobile Asides

### PR DESCRIPTION
**Pull request by aishwaryamathuria**
As per consonant the min-height in all in mobile view needs to be removed as long as all blocks show/maintain 80px padding/margin top and bottom. PR includes following changes -

Move min heights for different asides to tablet and desktop
if there is no foreground media or image in standard or no background or foreground media or image in split then change the gap for both to have 80px at border.

Resolves: [MWPW-135782](https://jira.corp.adobe.com/browse/MWPW-135782)

Test URLs:

Before: https://main--milo--adobecom.hlx.page/drafts/mathuria/aside/mwpw-135782?martech=off
After: https://asidemobile--milo--aishwaryamathuria.hlx.page/drafts/mathuria/aside/mwpw-135782?martech=off

### CODE REVIEW

File: libs/blocks/aside/aside.css [hunk 1]
```diff
@@ -504,7 +492,29 @@
   width: auto;
 }
 
+@media screen and (max-width: 600px) {
+  .aside.no-media:not(.notification) .foreground.container {
+    gap: 0;
+  }
+
+  .aside.split.no-media:not(.notification) .foreground.container .text {
+    padding: var(--spacing-xxxl) 0 var(--spacing-xxxl) 0;
```
Comment (Sartxi) [inline libs/blocks/aside/aside.css:501]:
```suggestion
    padding: var(--spacing-xxxl) 0;
```

**Review by Sartxi — APPROVED** [review]

**Comment by spadmasa** [comment]
Validation done on the below the url :
https://asidemobile--milo--aishwaryamathuria.hlx.page/drafts/mathuria/aside/mwpw-135782?martech=off
image
image

image

image

https://asidemobile--milo--aishwaryamathuria.hlx.page/drafts/mathuria/aside/mwpw-129917?martech=off
image

image

image

image

https://asidemobile--milo--aishwaryamathuria.hlx.page/docs/library/kitchen-sink/aside
image

image

https://asidemobile--milo--aishwaryamathuria.hlx.page/docs/library/kitchen-sink/aside-quiet
image
image

https://asidemobile--milo--aishwaryamathuria.hlx.page/docs/library/kitchen-sink/aside-split
image

File: libs/blocks/aside/aside.css (hunk 1 above)
Comment (aishwaryamathuria) [reply libs/blocks/aside/aside.css]: Made the suggested change in the PR

**Review by Ruchika4 — APPROVED** [review]
Looks Good

**Pull request merged**